### DISCUSSION TITLE
Where can a list of GitHub achievements be found?

### DISCUSSION THREAD

**Discussion by s-spoon** [community/community#50661] [Actions]
### Select Topic Area

Question

### Body

I want to make my GitHub look awesome. Are there some achievements that I can quickly get on GitHub?

I found this excellent [reference](https://github.com/Schweinepriester/github-profile-achievements) but it's not enough. What are all the achievements in GitHub and what is the criteria to achieve them?

**Comment by SuperJUNOed** [comment]
What achievements are there?

Great question, we’ll never tell 🤫

There’s a bit of whimsy, surprise, and delight that we hope you’ll feel upon discovering achievements on your own profile, as well as on your fellow contributors’ and maintainers’ profiles. You will be able to infer the criteria to unlock certain achievements by viewing an achievement you’ve earned, or by viewing someone else’s achievements.

**Comment by tuves** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
